ooiui/static/js/views/common/TroubleTicketView.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+/*
+* ooiui/static/js/views/common/TroubleTicketView.js
+*
+* Dependencies
+* Partials
+* - ooiui/static/js/partials/TroubleTicket.html
+*/
+var TroubleTicketView = Backbone.View.extend({
```

Comment:
@DanielJMaher you'll need an event handler here to capture the button click and save the model.

```
events: {
    'click .btn-sm' : 'onSubmit'
}
```